lib/spack/docs/packaging_guide.rst

```diff
@@ -3298,7 +3298,7 @@ follows:
        # The header provided by the bar virtual package
        @property
        def bar_headers(self):
-           return find_headers("bar/bar.h", root=self.home.include, recursive=False)
+           return find_headers("bar", root=self.home.include, recursive=False)
```

Comment:
Use `self.home.include.bar` here to show how to handle sub-directories like before.

Reply:
After writing this comment I ran into the GL issue. I think this is a case where filtering may be beneficial to show, but maybe better to fix the API.